client/src/components/Menu/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```

Comment:
rename this file into `Menu.js` and create package.json file by example from other components